bwip-js/index.d.ts

```diff
@@ -0,0 +1,83 @@
+// Type definitions for bwip-js 1.1.1
```

Comment:
Moved renames to their own commit. `git log --follow bwip-js/index.d.ts` should work now.